File: Components/DevExpressVCL/Demos/ExpressQuantumGrid/CBuilder/Common/CarsDataForGrid.cpp
//---------------------------------------------------------------------------

#include <vcl.h>
#include <stdlib.h>
#include "shellapi.hpp"
#pragma hdrstop

#include "CarsDataForGrid.h"
//---------------------------------------------------------------------------
#pragma package(smart_init)
#pragma link "cxLookAndFeels"
#pragma link "CarsData"
#pragma link "cxEditRepositoryItems"
#pragma link "cxDBExtLookupComboBox"
#pragma link "cxEdit"
#pragma link "cxDBEditRepository"
#pragma link "cxClasses" 
#pragma link "dxmdaset"
#pragma link "cxStyles"
#pragma link "cxCustomData" 
#pragma link "cxGraphics"
#pragma link "cxFilter"
#pragma link "cxData"
#pragma link "cxDataStorage"
#pragma link "cxNavigator"
#pragma link "cxDBData"
#pragma link "cxHyperLinkEdit"
#pragma link "cxGridCustomTableView"
#pragma link "cxGridTableView"
#pragma link "cxGridBandedTableView"
#pragma link "cxGridDBBandedTableView"
#pragma link "cxControls"
#pragma link "cxGridCustomView"
#pragma link "cxGrid"
#pragma resource "*.dfm"
TdmGridCars *dmGridCars;
//---------------------------------------------------------------------------
__fastcall TdmGridCars::TdmGridCars(TComponent* Owner): TdmCars(Owner)
{
}

File: Components/DevExpressVCL/Demos/ExpressQuantumGrid/Delphi/Common/CarsDataForGrid.pas
unit CarsDataForGrid;

{$I cxVer.inc}

interface

uses
  SysUtils, Classes, CarsData, cxEditRepositoryItems, cxDBExtLookupComboBox, cxEdit, cxDBEditRepository,
  cxClasses, DB, dxmdaset, cxStyles, cxCustomData, cxGraphics, cxFilter, cxData, cxDataStorage, cxNavigator,
  cxDBData, cxHyperLinkEdit, cxGridCustomTableView, cxGridTableView, cxGridBandedTableView, cxGridDBBandedTableView,
  cxControls, cxGridCustomView, cxGrid;

type

  { TdmGridCars }

  TdmGridCars = class(TdmCars)
    EditRepositoryTrademarkLookup: TcxEditRepositoryExtLookupComboBoxItem;
    GridViewRepository: TcxGridViewRepository;
    GridViewRepositoryDBBandedTableView: TcxGridDBBandedTableView;
    GridViewRepositoryDBBandedTableViewDescription: TcxGridDBBandedColumn;
    GridViewRepositoryDBBandedTableViewID: TcxGridDBBandedColumn;
    GridViewRepositoryDBBandedTableViewLogo: TcxGridDBBandedColumn;
    GridViewRepositoryDBBandedTableViewName: TcxGridDBBandedColumn;
    GridViewRepositoryDBBandedTableViewRecId: TcxGridDBBandedColumn;
    GridViewRepositoryDBBandedTableViewSite: TcxGridDBBandedColumn;
  end;

var
  dmGridCars: TdmGridCars;

implementation

{%CLASSGROUP 'Vcl.Controls.TControl'}

{$R *.dfm}

end.
